pkg/apis/psmdb/v1/psmdb_types.go

```diff
@@ -549,12 +549,17 @@ func (cr *PerconaServerMongoDB) CompareVersion(version string) int {
 	return cr.Version().Compare(v.Must(v.NewVersion(version)))
 }
 
-func UserSecretNameInternal(cr *PerconaServerMongoDB) string {
-	internalPrefix := "internal-"
+const internalPrefix = "internal-"
+const userPostfix = "-users"
 
+func InternalSecretName(cr *PerconaServerMongoDB) string {
```

Comment:
```suggestion
func InternalUserSecretName(cr *PerconaServerMongoDB) string {
```
not clear it is ssl or users internal secret

Reply:
fixed in https://github.com/percona/percona-server-mongodb-operator/pull/599/commits/f8b55dda111811ca6860520dfc43ead2e16b4618